src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -397,4 +400,95 @@ describe('buildSettings', () => {
       );
     });
   });
+
+  describe('OnDemandTv', () => {
+    const hindiTvBaseSettings = {
+      counterName: 'hindi.bbc_hindi_tv.tv.w172zm8920nck2z.page',
+      lang: 'hi',
+      service: 'hindi' as Services,
+      statsDestination: 'WS_NEWS_LANGUAGES',
+      producer: 'HINDI',
+      translations: hindiServiceConfig.default.translations,
+    } as BuildConfigProps;
+
+    const hindiTvMediaBlocks = hindiTvProgramme.content.blocks.map(
+      tvMediaBlock => {
+        return {
+          type: 'tvMedia',
+          model: {
+            ...tvMediaBlock,
+          },
+        };
+      },
+    );
+    it('Should process a On Demand TV block into a valid playlist item.', () => {
+      const result = buildSettings({
+        ...hindiTvBaseSettings,
+        blocks: hindiTvMediaBlocks as MediaBlock[],
+        pageType: MEDIA_PAGE,
+      });
+
+      expect(result).toStrictEqual({
+        playerConfig: {
+          product: 'news',
+          enableToucan: true,
+          appType: 'responsive',
+          autoplay: false,
+          externalEmbedUrl: '',
+          mediator: { host: 'open.test.bbc.co.uk' },
+          appName: 'news-hindi',
+          counterName: 'hindi.bbc_hindi_tv.tv.w172zm8920nck2z.page',
+          statsObject: {
+            destination: 'WS_NEWS_LANGUAGES',
+            producer: 'HINDI',
+            episodePID: 'w172zm8920nck2z',
+          },
+          ui: {
+            controls: { enabled: true },
+            fullscreen: { enabled: true },
+            locale: {
+              lang: 'hi',
+            },
+            subtitles: {
+              defaultOn: true,
+              enabled: true,
+            },
+          },
```

Comment:
We might need to check with product whether these default UI settings are acceptable, or whether they need to be updated/removed. cc @gavinspence